geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/internal/TestConfigService.java

```diff
@@ -26,24 +26,24 @@
  * Generates fake JdbcConnectorService with Connections and RegionMappings for tests.
  */
 public class TestConfigService {
-  private static final String DB_NAME = "DerbyDB";
+  // private static final String DB_NAME = "DerbyDB";
```

Comment:
Did you want to delete these commented-out lines?

Reply:
Yes, I will push an additional commit to clean up some of these lines.